.github/workflows/publish_release.yml

```diff
@@ -0,0 +1,170 @@
+name: Publish NovelRT
+
+on:
+  workflow_dispatch:
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+
+    outputs:
+      tag: ${{ steps.tag_version.outputs.new_tag }}
+      id: ${{ steps.create_release.outputs.id }}
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: mathieudutour/github-tag-action@v6.0
+      id: tag_version
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+
+    - uses: ncipollo/release-action@v1
+      id: create_release
+      with:
+        tag: ${{ steps.tag_version.outputs.new_tag }}
+        name: NovelRT ${{ steps.tag_version.outputs.new_tag }}
+        body: ${{ steps.tag_version.outputs.changelog }}
+        prerelease: true
+        draft: true
+
+  build-release:
+    needs: create-release
+    runs-on: ${{ matrix.os }}
+    container: ${{ matrix.container }}
+
+    strategy:
+      matrix:
+        configuration: [ RelWithDebInfo, MinSizeRel ]
+        os: [ ubuntu-latest, windows-latest, macos-10.15 ]
+        exclude:
+        - os: macos-10.15
+          configuration: RelWithDebInfo
+        - os: macos-10.15
+          configuration: MinSizeRel
+        include:
+        - os: ubuntu-latest
+          container: novelrt/novelrt-build:latest
+          profile: linux-clang10-amd64
+          name: Ubuntu
+        - os: macos-10.15
+          profile: macos-appleclang12-amd64
+          configuration: Release
+          name: macOS
+        - os: macos-10.15
+          profile: macos-appleclang12-amd64
+          configuration: Debug
+          name: macOS
+        - os: windows-latest
+          profile: windows-vs2022-amd64
+          name: Windows
+    steps:
+    - uses: actions/checkout@v2
+    - run: git fetch --depth=1 origin +refs/tags/*:refs/tags/*
+
+    - name: Prepare build environment
+      run: |
+          pip3 install conan
+          conan config install https://github.com/NovelRT/ConanConfig.git
+
+    - name: Generate Build System
+      run: conan install . -if build --build=missing --profile ${{ matrix.profile }}-${{ matrix.configuration }} -o novelrt:buildsamples=False -o novelrt:buildinterop=False -o novelrt:documentation=False
+
+    - name: Build NovelRT
+      run: conan build . --build-folder build
+
+    - name: Install NovelRT - ${{ matrix.configuration }}
+      run: cmake --install build --config "${{ matrix.configuration }}"
+
+    - name: Copy Third Party Artifacts and Licence (Windows)
+      run: |
+        Get-ChildItem build/thirdparty -Filter *.dll | Copy-Item -Destination build/package/bin -Force -PassThru
+        Get-ChildItem . -Filter LICENCE-DIST.md | Copy-Item -Destination build/package/bin/LICENCE.md -Force -PassThru
+      if: "contains(matrix.os, 'windows')"
+
+    - name: Copy Third Party Artifacts and Licence (macOS)
+      run: |
+          cp -v -f build/thirdparty/*.dylib build/package/lib
+          cp -v -f build/thirdparty/*.json build/package/bin
+          cp -v -f ./LICENCE-DIST.md build/package/bin/LICENCE.md
```

Comment:
This file is essentially all licences for binaries that are redistributed and require attribution + NovelRT's licence. Its only copied for macOS and Windows due to the third party dependency distribution - so Linux is exempt from needing this IIRC.

Reply:
We should probably copy the file on Linux too just for safety's sake.

Reply:
Makes sense - I'll switch the Linux one to copying LICENCE-DIST.md instead of just the NovelRT licence